dissect/target/plugins/browsers/chromium.py

```diff
@@ -23,76 +25,86 @@ class ChromiumMixin:
     """Mixin class with methods for Chromium-based browsers."""
 
     DIRS = []
-    BrowserHistoryRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
-        "browser/chromium/history", GENERIC_HISTORY_RECORD_FIELDS
-    )
     BrowserDownloadRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
         "browser/chromium/download", GENERIC_DOWNLOAD_RECORD_FIELDS
     )
+    BrowserExtensionRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
+        "browser/chromium/extension", GENERIC_EXTENSION_RECORD_FIELDS
+    )
+    BrowserHistoryRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
+        "browser/chromium/history", GENERIC_HISTORY_RECORD_FIELDS
+    )
 
-    def history(self, browser_name: str = None) -> Iterator[BrowserHistoryRecord]:
-        """Return browser history records from supported Chromium-based browsers.
+    def _build_userdirs(self, hist_paths: list[str]) -> list[tuple]:
+        """Join the selected browser dirs with the user home path.
 
         Args:
-            browser_name: The name of the browser as a string.
+            hist_paths: A list with browser paths as strings.
+
+        Returns:
+            list with tuples containing user and TargetPath object.
+        """
+        users_dirs: list[tuple] = []
+        for user_details in self.target.user_details.all_with_home():
+            for d in hist_paths:
+                cur_dir: TargetPath = user_details.home_path.joinpath(d)
+                if not cur_dir.exists():
+                    continue
+                users_dirs.append((user_details.user, cur_dir))
+        return users_dirs
+
+    def _iter_db(self, filename: str) -> Iterator[SQLite3]:
+        """Generate a connection to a sqlite history database file.
```

Comment:
There's no "connection", it's a parsed database. So "an opened SQLite database file" would be more accurate I think.